src/store/flow/flow.ts

```diff
@@ -496,33 +498,30 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
     })
 
     // add missing variants
```

Comment:
👇 extract `resources.filter(...)` as `nonChoiceResources` and reuse it. No other changes